neuralprophet/forecaster.py

```diff
@@ -199,13 +199,10 @@ class NeuralProphet:
                 * ``shared_neural_nets``
                 * ``shared_neural_nets_coef``
 
-        future_regressors_d_hidden: int
-            Number of hidden layers in the neural network model for future regressors.
-            Ignored if ``future_regressors_model`` is ``linear``.
+        future_regressors_layers: list of int
+            array of hidden layer dimensions of the future regressor nets. Specifies number of hidden layers (number of entries)
```

Comment:
to avoid confusion, phase `array of..` as `list of ...`
Should also state default values.
Looks like we currently have [4,4]. We might want roll this back to no hidden layers by default. Thoughts?